### ISSUE TITLE
Oracle support

### ISSUE THREAD

**Issue by xxxEWxxx** [F43nd1r/Acrarium#9]
It would be great is Oracle could be used as database. I've tried it but there are at least these issues that prevent it:

- there is more than one primary key with name "PRIMARY"
- "user" as tablename must be escaped in generated SQLs
- "level" as columnname must be escaped in generated SQLs
- some generated SQLs don't work because of Oracle is case-sensitiv

**Comment by F43nd1r** [comment]
Are you using the correct dialect for your database? e.g.
```properties
spring.jpa.database-platform=org.hibernate.dialect.Oracle12cDialect
```

I would expect hibernate to automatically handle escaping if the correct dialect is used.

**Comment by xxxEWxxx** [comment]
Yes I'm quite sure with the dialect. Did you managed to run the liquibase updates on an empty oracle DB?

**Comment by F43nd1r** [comment]
No, I don't have oracle db installed. Will check in the coming days.

**Comment by xxxEWxxx** [comment]
Thank you! I'm using this image https://hub.docker.com/r/sath89/oracle-12c/

**Comment by F43nd1r** [comment]
Unfortunately, oracle support is blocked by a [hibernate issue](https://hibernate.atlassian.net/browse/HHH-1615). It is unlikely that this issue is resolved before the bug in hibernate is fixed.

**Comment by F43nd1r** [comment]
After some consideration I've changed my mind on this: mysql will be the only dialect 1.0.0 supports officially. Others might or might not work. Oracle doesn't work and probably won't ever, unless someone else comes and does the work.
